script.js

```diff
@@ -1,15 +1,106 @@
-const modal = document.querySelector('#instructionModal');
-const closeBtn = document.querySelector('.close');
+const instructionModal = document.querySelector('#instructionModal');
+const instructionCloseBtn = document.querySelector('.instructionClose');
 const instructionsButton = document.querySelector('#instructionsButton')
+const startButton = document.querySelector('.start')
+const gameOverModal = document.querySelector('#gameOverModal');
+const gameOverCloseBtn = document.querySelector('.gameOverClose');
 
-closeBtn.addEventListener('click', () => {
-    modal.classList.remove('open');
-});
 
 
-const openInstructionsModal = () => {
+const openModal = (modal) => {
     modal.classList.add('open');
 }
 
-instructionsButton.addEventListener('click', openInstructionsModal);
+const closeModal = (modal) => {
+    modal.classList.remove('open');
+}
+
+instructionsButton.addEventListener('click', () => {
+    openModal(instructionModal)
+})
+
+instructionCloseBtn.addEventListener('click', () => {
+    closeModal(instructionModal);
+})
+
+gameOverCloseBtn.addEventListener('click', () => {
+    closeModal(gameOverModal);
+})
+
+const gameOver = () => {
+    openModal(gameOverModal)
+}
+// max's get pattern functions:
+
+// const
+let pattern = []
+function getRandBoxes(boxCount) {
+    let newArray = [];
+    for (let i = 0; i < boxCount; i++) {
+        newArray.push(Math.floor(Math.random() * 9))
+    }
+    pattern = newArray;
+}
+
+const box1 = document.querySelector('.box1');
+const box2 = document.querySelector('.box2');
+const box3 = document.querySelector('.box3');
+const box4 = document.querySelector('.box4');
+const box5 = document.querySelector('.box5');
+const box6 = document.querySelector('.box6');
+const box7 = document.querySelector('.box7');
+const box8 = document.querySelector('.box8');
+const box9 = document.querySelector('.box9');
```

Comment:
do these actually do anything other than go straight into the array below? if not, why not give them all the same clas sin the html then use `querySelectorAll`, that will give you an array of all boxes in 1 line of code